components/MobileFilterModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { Fab, Grid } from '@material-ui/core/'
+import { useStyles } from '../constants'
+import { CategoryFilters } from '../components'
+import { Filter } from '../types/filter'
+const MobileFilterModal = ({
+  citySelected,
+  serviceSelected,
+  genderSelected,
+  languageSelected,
+  handleSelected,
+  MenuProps,
+  routeCategory,
+  modalStyle,
+  setOpen,
+  activeCopy,
+}: Filter) => {
+  const classes = useStyles()
+  return (
+    <>
+      <Grid
+        container
+        spacing={3}
+        style={modalStyle}
+        className={classes.paperFilter}
+      >
+        <Grid item xs={12} sm={12} md={3} style={{ marginLeft: '2rem' }}>
```

Comment:
I think you're overcomplicating things with all the absolute values here. could you do this with `display:flex`,`justify-content:center` and `align-items:center`?